examples/01-main-example/sample

```diff
@@ -1,5 +1,9 @@
 #!/usr/bin/env bash
 
+DMG_FILE_NAME=Application-Installer.dmg
+APP_NAME="Application"
+PATH_TO_PARENT_OF_APP="source_folder/"
```

Comment:
This probably should be renamed to `PATH_TO_SOURCE_FILES`

Reply:
> `PATH_TO_PARENT_OF_APP` might be renamed to `PATH_TO_SOURCE_FILES` or similar

Use `SOURCE_FOLDER_PATH`? README says:

> Usage
> -----
> 
> ```sh
> create-dmg [options ...] <output_name.dmg> <source_folder>
> ```